DESCRIPTION

```diff
@@ -21,7 +21,8 @@ Authors@R: c(
     person("Jojo", "Miller", role = "ctb"),
     person("Jayla", "Nakayama", role = "ctb"),
     person("Natalia", "Iannucci", email = "iannuccinatalia@gmail.com", role = "ctb"),
-    person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb"))
+    person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb")),
```

Comment:
Hey @sunniraleigh, the mismatched parentheses is causing the package build to fail. Please be sure to both:
* Build the package
* Run the package "check"
and ensure all errors are resolved before making any commits

Reply:
@beanumber